jsonnet/rules.jsonnet

```diff
@@ -196,6 +196,17 @@
               severity: 'warning',
             },
           },
+          {
+            expr: 'sum(max by(namespace, container, pod) (increase(kube_pod_container_status_restarts_total[12m])) and max by(namespace, container, pod) (kube_pod_container_status_last_terminated_reason{reason="OOMKilled"}) == 1) > 5',
```

Comment:
iirc 12min comes from some kube behavior right? Do we have some docs about this to link to for future Frederic to understand?

Reply:
ha :D yes this is the way we restart containers, no docs just a lot of testing with various numbers of timespans, so it eventually normalised the graph.

Reply:
ok, that works for me .. no need for comment then